components/Loading/samples/LoadingSample.xaml

```diff
@@ -0,0 +1,25 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="LoadingExperiment.Samples.LoadingSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:CommunityToolkit.WinUI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:LoadingExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+      mc:Ignorable="d">
+
+    <Grid>
+        <controls:Loading IsLoading="{x:Bind LoadingState, Mode=OneWay}">
```

Comment:
Maybe as another sample we can combine with the MVVM Toolkit and the Converter Task helper we have for the `TaskResultConverter` so that the loading command/task thing auto sets the thing to true? (Just have a dummy delay for like 5 seconds?)